libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -266,30 +273,35 @@ void AP_MotorsMatrix::output_armed_stabilizing()
 
     // calculate amount of yaw we can fit into the throttle range
     // this is always equal to or less than the requested yaw from the pilot or rate controller
-    float rp_low = 1.0f;    // lowest thrust value
     float rp_high = -1.0f;  // highest thrust value
-    for (i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
-        if (motor_enabled[i]) {
-            // calculate the thrust outputs for roll and pitch
-            _thrust_rpyt_out[i] = roll_thrust * _roll_factor[i] + pitch_thrust * _pitch_factor[i];
-            // record lowest roll + pitch command
-            if (_thrust_rpyt_out[i] < rp_low) {
-                rp_low = _thrust_rpyt_out[i];
-            }
-            // record highest roll + pitch command
-            if (_thrust_rpyt_out[i] > rp_high && (!_thrust_boost || i != _motor_lost_index)) {
-                rp_high = _thrust_rpyt_out[i];
-            }
+    float yaw_allowed = 1.0f; // amount of yaw we can fit in
+    for (uint8_t i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
+        if (!motor_enabled[i]) {
```

Comment:
What is the advantage of the pattern:
 if (!motor_enabled[i]) {
    continue;
}

over:
 if (motor_enabled[i]) {
     bla bla
}

Reply:
It keeps the if small and makes it clear that we don't do anything if not enabled (there is can be nothing after the if and still in the for loop). Also means that the key code is only in a single loop. 

More of a personal preference thing. I have removed. Its not so bad now in any case as we have removed `rp_high` and `rp_low` so the if's are smaller.

Reply:
Thanks